systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationST.java

```diff
@@ -671,11 +671,11 @@ void setUp()  {
 
         resourceManager.createResourceWithWait(
             NodePoolsConverter.convertNodePoolsIfNeeded(
-                KafkaNodePoolTemplates.brokerPool(Environment.TEST_SUITE_NAMESPACE, KafkaNodePoolResource.getBrokerPoolName(oauthClusterName), oauthClusterName, 1).build(),
-                KafkaNodePoolTemplates.controllerPool(Environment.TEST_SUITE_NAMESPACE, KafkaNodePoolResource.getControllerPoolName(oauthClusterName), oauthClusterName, 1).build()
+                KafkaNodePoolTemplates.brokerPoolPersistentStorage(Environment.TEST_SUITE_NAMESPACE, KafkaNodePoolResource.getBrokerPoolName(oauthClusterName), oauthClusterName, 1).build(),
```

Comment:
You are changing this, but you should increase the number of replicas to 3/3 just for the rolling update, or do you think that the persistent storage will be sufficient? That's for the `testSuperUserWithOauthAuthorization`